easybuild/easyconfigs/l/liknorm/liknorm-1.4.6-foss-2018b.eb

```diff
@@ -0,0 +1,23 @@
+
+easyblock = 'CMakeMake'
+
+name = 'liknorm'
+version = '1.4.6'
+
+homepage = 'https://github.com/limix/liknorm'
+description = """Moments of the product of an exponential-family likelihood with a Normal distribution."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
Since this software does not need `MPI`, `ScaLAPACK` or `FFTW` but performance can be affected by the compiler used to build it, I would suggest building it with the `GCC-7.3.0-2.3` toolchain.

I would also add:
```python
toolchainopts = {'pic': True}
```